iframify.js

```diff
@@ -149,19 +65,25 @@
     var iframe = document.createElement('iframe');
     var html = getIframeContentForNode(node, extraCSS);
     iframe.srcdoc = html;
-    iframe.src = 'data:text/html;charset=utf-8,' + encodeURI(html);
-    node.parentNode.replaceChild(iframe, node);
 
-    // If `srcdoc` is supported, there is no CORS issue therefore we can
-    // dynamically resize iframe height. The setTimeout is there to make sure
-    // that any asynchronously loaded content inside the component has been
-    // safely loaded before computing the height.
-    if ('srcdoc' in iframe) {
-      setTimeout(function () {
-        var iframeDocument = iframe.contentDocument || iframe.contentWindow.document;
-        iframe.height = getDocumentHeight(iframeDocument);
-      }, 500);
+    if (!('srcdoc' in iframe)) {
+      console.log(
+        'Your browser does not support the `srcdoc` attribute on <iframe>.' +
+        'Therefore, it is not possible to wrap node in an iframe because of' +
+        ' CORS policy. :('
+      )
+
+      return false;
```

Comment:
Given that we return the iframe when it’s working correctly, shall we return `null` there to be sync with `querySelector(..)` return values?